### PR TITLE
translate nodejs concepts challenge to english

### PR DESCRIPTION
**Pull request by LauraBeatris**
Hey, what's up 😃

I just added the English version of the Node.js Concepts Challenge. 
There may be some typos yet, so a review would be awesome 🚀

### CODE REVIEW

File: desafio-conceitos-nodejs/README.en.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-<img alt="GoStack" src="https://storage.googleapis.com/golden-wind/bootcamp-gostack/header-desafios.png" />
+<img alt="GoStack" src="https://storage.googleapis.com/golden-wind/bootcamp-gostack/header-desafios.png" style="width: 100%;"/>
 
 <h3 align="center">
   Challenge 02: Node.js Concepts
 </h3>
 
-<p align="center">“Don't wait to to plant, only have patient to harvest”!</blockquote>
+<p align="center">“Don't wait to plant, only have patient to harvest”!</blockquote>
```
Comment (josepholiveira) [inline desafio-conceitos-nodejs/README.en.md:7]: Change patient for patience

File: desafio-conceitos-nodejs/README.en.md [hunk 2]
```diff
@@ -21,92 +21,93 @@
...
+So why not use `PUT` instead of `POST`? Precisely because we are "creating" A new like, and not updating the number of likes to any other value.
 
-Talvez fique difícil enxergar por ser apenas um número, mas pense que cada like seja salvo em uma tabela no banco junto do usuário que realizou esse like. Agora fica mais claro que você está criando um novo like, certo?
+It may be difficult to see because it is just a number, but think that each like is saved in a table in the database with the user who added this like. Now it’s clearer that you’re creating a new like, right?
 
-Bons estudos <3
+Good studies <3
 
-### Específicação dos testes
+### Testing Specification
```
[the remark this answers is not in the document]
Comment (LauraBeatris) [reply desafio-conceitos-nodejs/README.en.md:75]: Good catch! It's not verb so there's not need to use "ing"

File: desafio-conceitos-nodejs/README.en.md (hunk 1 above)
Comment (LauraBeatris) [reply desafio-conceitos-nodejs/README.en.md:7]: Good catch! It should be a subjective instead of a adjetive

**Comment by josepholiveira** [comment]
Well, guess i have to approve it now.

Thank you, @LauraBeatris! 🚀 💜

**Pull request merged**